projects/discussion-ui/src/lib/components/discuss-start/discuss-start.component.ts

```diff
@@ -67,13 +67,19 @@ export class DiscussStartComponent implements OnInit {
       const tags = _.map(_.get(topicData, 'tags'), (element) => {
         return _.get(element, 'value');
       });
-      this.startForm.controls['question'].setValue(_.get(topicData, 'title'));
+      this.startForm.controls['question'].setValue(this.htmlDecode(_.get(topicData, 'title')));
       this.startForm.controls['description'].setValue(_.get(topicData, 'posts[0].content').replace(/<[^>]+>/g, ''));
       this.startForm.controls['tags'].setValue(tags);
       this.validateForm();
     }
   }
 
+  /** The htmlDecode() method parses a string containing either HTML or XML */
```

Comment:
JSDOC format is not correct.

Is this method only for this component. Can't this method can be used by other components? 
If other components can use, then it should be in util service.

Reply:
This follows the
 SRP principle -> This is not related to this component.
 DRY principle -> If other components requires the same method they have to copy-paste the same in another component
 
 @christyfernandes @sudip-mukherjee 

This mehtod is